chapters/appendix.adoc

```diff
@@ -55,38 +55,45 @@ Chapter <<Scoring Goals>>:
 | Invalid Goal | <<Ball In And Out Of Play, ball in play>> | <<Stop>>, then <<Free Kick>> | icon:check[role="green"]
 |===
 
-Chapter <<Offenses>>, section <<Minor Offenses>>:
+Chapter <<Rule Violations>>, section <<Stopping Offenses>>:
```

Comment:
```suggestion
Chapter <<Rule Violations>>, section <<Non Stopping Offenses>>:
```

Reply:
I guess I misread this section. I do now get the difference between Stopping Offenses and Stopping Fouls.

I'm not sure about Lack of progress here: Why is it listed here?

Reply:
I've refactored the Stopping Offenses away